Tella/Scenes/Resources/ViewModel/AvailableResourcesVM.swift

```diff
@@ -0,0 +1,108 @@
+//
+//  AvailableResourcesVM.swift
+//  Tella
+//
+//  Created by gus valbuena on 2/5/24.
+//  Copyright © 2024 HORIZONTAL. All rights reserved.
+//
+
+import Combine
+import Foundation
+import Combine
+
+class AvailableResourcesVM: ObservableObject {
+    @Published var appModel: MainAppModel
+    @Published var availableResources: [ResourceCardViewModel] = []
+    @Published var downloadedResourcesVM: DownloadedResourcesVM?
+    @Published var isLoadingList: Bool = false
+    @Published var isLoadingDownload: Bool = false
+    @Published var servers: [TellaServer] = []
```

Comment:
I think servers  is not used externally and we don't need it to be published 
`private var servers: [TellaServer] = []`